routes/index.js

```diff
@@ -164,12 +164,15 @@ var rootPost = function(req, res, next) {
 
   var ref = (req.headers && req.headers.referer) || false;
   //No POST request allowed from other sources
+  //TODO probably need to fix this for prod docker
   if (ref) {
     var u = url.parse(ref);
     var hostname = u && u.hostname.toLowerCase();
+    var environment = env.toLowerCase();
     if (hostname === "kaha.co" ||
       hostname === "demokaha.herokuapp.com" ||
-      env.toLowerCase() === "stage"
+      environment === "stage" ||
+      environment === "dev"
```

Comment:
You removed it here: https://github.com/kahaco/kaha/commit/c79f01ca7936c900ec2b3d6729e67234979cecf5

I added this back on purpose so that POST works on local redis installations (with/without docker). Why was it removed in the first place?